src/test/java/pl/pop/interview/master/answer/AnswerServiceTest.java

```diff
@@ -40,51 +40,51 @@ public void testFindRandomQuestion() {
 
     @Test
     public void testSaveNewCorrectAnswer() {
-        Question question1 = new Question("Question1Content", YesNo.YES);
+        Question question1 = new Question("Question1Content", "Yes");
```

Comment:
ogólnie spoko ale na przyszłość: osobiście jakbym już miał to poprawiać to bym zrobił stałe w polach
 private final String YES = "Yes" (analogicznie dla "No") i ich użył w testach zamiast wszędzie Stringa pisać z ręki